### PR TITLE
UBO-166 Limit number of authors to 15

### PR DESCRIPTION
**Pull request by Possommi**

### CODE REVIEW

File: ubo-common/src/main/resources/xsl/mods-display.xsl [hunk 1]
```diff
@@ -313,15 +335,24 @@
 
   <!-- ========== Personennamen als Liste ========== -->
   <xsl:template match="mods:name" mode="brief">
```
Comment (Possommi) [inline ubo-common/src/main/resources/xsl/mods-display.xsl:337]: @kkrebs one change here

File: ubo-common/src/main/resources/xsl/mods-display.xsl [hunk 2]
```diff
@@ -330,45 +361,62 @@
...
+    <xsl:variable name="list" select="../mods:name[mods:role/mods:roleTerm[@type='code']=$role]"/>
     <xsl:if test="count($list[1]|.)=1">
       <div class="row">
         <div class="col-3">
-          <xsl:apply-templates select="$role" />
+          <xsl:apply-templates select="$role"/>
           <xsl:text>:</xsl:text>
         </div>
         <div class="col-9">
           <xsl:for-each select="$list">
```
Comment (Possommi) [inline ubo-common/src/main/resources/xsl/mods-display.xsl:395]: @kkrebs the other one here

**Review by kkrebs — APPROVED** [review]
looks good for me, but should also be checked from kerstin

File: ubo-common/src/main/resources/config/ubo-common/messages_de.properties [hunk 3]
```diff
@@ -113,6 +113,8 @@ button.statistics         = Statistik
 button.statisticsShow     = Statistik anzeigen
 button.submit             = Abschicken
 button.upload             = Dateien hochladen
+button.hide.authors       = {0} Autoren verbergen
```
Comment (kkrebs) [inline ubo-common/src/main/resources/config/ubo-common/messages_de.properties:116]: english translation is missing

Comment (Possommi) [reply ubo-common/src/main/resources/config/ubo-common/messages_de.properties:116]: mea culpa

**Review by kkrebs — APPROVED** [review]

**Pull request merged**